### DISCUSSION TITLE
Calculating radial dipole matrix elements

### DISCUSSION THREAD

**Discussion by deejaybee11** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#181] [Q&A]
I am wondering why there is such a significant difference between the literature values for the radial DME and the calculated one. When calculating the radial DME for the D2 transition of Rb87 I get the value 5.57 when calculating the integral and -5.18 when taking the literature value. This is a rather large discrepancy, why is this so?

**Comment by nikolasibalic** [comment]
Thank you for your question @deejaybee11 and apologies for late reply.

ARC will return literature value automatically, since it uses those when existing, as the package is "aware" of limitations and has simple heuristics to decide what to return.

The discrepancy is expected compared to simple integration of wave function since approximations and model potential approach don't work that well for low-lying states. There are more complicated and accurate calculations for low lying states that can be accessed for example [here](https://www1.udel.edu/atom/).
